src/styles/card.css

```diff
@@ -0,0 +1,127 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 12px;
+  width: 200px;
+  border: 1px solid var(--color-gray);
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.card__image {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 15px;
+}
+
+.card__image > img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.card__description {
+  margin-bottom: 6px;
+}
+
+.card__title {
+  margin: 25px auto 0;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  line-height: 18px;
+  font-style: normal;
+  font-size: 12px;
+  margin-bottom: 1px;
+}
+
+.card__subtitle {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  color: var(--color-secondary);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  font-style: normal;
+}
+
+.card__rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: 10px;
+}
+
+.card__reviews {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  color: var(--color-primary);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  font-style: normal;
+  margin-bottom: 3px;
+}
+
+.card__rating-stars {
+  margin-top: 8px;
+  margin-bottom: 13px;
+  margin-left: 1px;
```

Comment:
I don't think you need so many margins
Set just bottom one and it should work
Otherwise you did something else wrong